<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>803</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>[ AvailData ] Signup Page - Nice to meet you!</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
background-image: url(C:/Users/deadb/Documents/GitHub/AvailData-Public/Client/login_photos/signup_background.jpg);
background-position: center;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>402</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family: Trebuchet MS;
font-size:22px;
}

QFrame
{
background:rgba(0,0,0,0.8);
border-radius:30px;
}

QTextEdit
{
background-color: transparent;
color: transparent;
}


QLabel
{
background-color: transparent;
color: rgb(0, 255, 0);
font-size:60px;
}

QLabel:hover
{
color: transparent;
background-image: url(&quot;C:/Users/deadb/Documents/GitHub/AvailData/login_page/hidden_masthead.png&quot;);
}


QLineEdit
{
background-color: rgb(49, 51, 56);
border-radius: 15;
color: rgb(0, 255, 0);
}
QLineEdit:hover
{
background-color: rgb(88, 91, 100);
border-radius: 15;
color: rgb(0, 255, 0);
font-size:26px;
}


QPushButton
{
color: white;
border-radius:15px;
background:  rgba(255,0,0,0.6);
font-size:20px;
}
QPushButton:hover
{
color: white;
border-radius:15px;
background: red;
font-size:22px;
}


QToolButton
{
background: red;
border-radius:60px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="submit_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>340</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_line_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_line_edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Strong Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="confirm_password_line_edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Confirm Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_line_edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hidden_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>360</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: transparent;
color: rgb(0, 255, 0);
font-size: 55px;
}</string>
      </property>
      <property name="text">
       <string>[ Sign Up ]</string>
      </property>
     </widget>
     <widget class="QPushButton" name="return_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
